param_namespace:
  # planning
  euclidean_distance_cutoff: 2.0 # [m]
  safe_obstacle_distance: 1.5 # [m]
  unknown_is_occupied: false
  navigation_tolerance: 0.5 # [m]
  min_altitude: 1.0 # [m]
  distance_penalty: 0.5
  greedy_penalty: 1.0
  vertical_penalty: 2.0
  planning_tree_resolution: 0.4
  max_waypoint_distance: 2.0 # [m]
  planning_timeout: 3.5 # [s]

  # visualization
  visualize_planner: true
  show_unoccupied: false
  tree_points_scale: 0.22
  field_points_scale: 0.22
  expansions_points_scale: 0.15
  path_points_scale: 0.18
  goal_points_scale : 0.28
